synapse/models/material.py

```diff
@@ -31,37 +31,88 @@ class MatModule(s_module.CoreModule):
 
     def getModelDefs(self):
         modl = {
+
+            'interfaces': (
+
+                ('phys:object', {
```

Comment:
Do we want to include weight here?

Reply:
It is included. `:mass` is the more accurate term than weight.